apps/settings/js/panels/about_more_info/panel.js

```diff
@@ -30,7 +30,7 @@ define(function(require) {
 
         hardwareInfo.init({
           deviceInfoMac: panel.querySelector('[data-name="deviceinfo.mac"]'),
-          fields: panel.querySelectorAll('[data-name="deviceinfo.bt_address"]')
+          fields: panel.querySelector('[data-name="deviceinfo.bt_address"]')
```

Comment:
you should rename the element to `btAddress`, and also rename the element name in test to pass the tests https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/test/unit/panels/about_more_info/hardware_info_test.js#L26